package.json

```diff
@@ -14,7 +14,7 @@
     "build:es5": "NODE_ENV=es2015 babel src -d dist/es2015",
     "build": "rm -Rf ./dist && yarn build:es5 && yarn build:cjs",
     "test": "npm run test:pick -- '_tests/**/*spec.js'",
-    "test:pick": "NODE_ENV=cjs mocha --require @babel/register --require jsdom-global/register --require _tests/spinup/scaffolding ",
+    "test:pick": "NODE_ENV=cjs mocha --require @babel/register --require jsdom-global/register --require _tests/spinup/scaffolding --exit",
```

Comment:
this fixes the following issue with yarn test just being stuck and timing out after completing the tests.
![image](https://user-images.githubusercontent.com/85158673/145818867-c0a01065-593d-4c26-8e74-fd7a52692117.png)

Reply:
OH, I was suffering from this for quite a while and was going to replace mocha by (a more common for me) jest next week.
Thank you!